packages/xs-extra/xapi-plugin.master/opam

```diff
@@ -9,6 +9,8 @@ remove: [
   ["ocamlfind" "remove" "worker"]
 ]
 depends: [
-  "obuild" {build}
+  "oasis" {build}
+  "ocamlbuild" {build}
+  "base-unix"
   "rpc"
```

Comment:
We could have `"rpc" {>= "1.9.51"}` here, but since it probably won't be upstreamed, it's not important to add this constraint.